suse/build/SPECS/jenkins.spec

```diff
@@ -71,10 +71,10 @@ rm -rf "%{buildroot}"
 %__install -D -m0755 "%{SOURCE7}" "%{buildroot}%{_datadir}/%{name}/migrate"
 
 %pre
-/usr/sbin/groupadd -r %{name} &>/dev/null || :
+/usr/bin/getent group %{name} &>/dev/null || /usr/sbin/groupadd -r %{name} &>/dev/null
 # SUSE version had -o here, but in Fedora -o isn't allowed without -u
-/usr/sbin/useradd -g %{name} -s /bin/false -r -c "@@SUMMARY@@" \
-	-d "%{workdir}" %{name} &>/dev/null || :
+/usr/bin/getent passwd %{name} &>/dev/null || /usr/sbin/useradd -g %{name} -s /bin/false -r -c "@@SUMMARY@@" \
```

Comment:
this is the recommended method for adding a user depending on if it exists or not